.github/PULL_REQUEST_TEMPLATE/pull_request_template_1.md

```diff
@@ -0,0 +1,15 @@
+## Describe The Outcome
+What does merging this PR into main achieve?
+
+## Dependency Changes
+If any depdency changes, list them here.
+
+## Before Screenshots (If applicable)
```

Comment:
Suggesting this alternative that's more readable as it makes better use of horizontal space.

| Before | After |
|:-:|:-:|
| ![image](https://github.com/user-attachments/assets/2e17e42e-42b8-46c6-9fa1-a14819f71393) | ![image](https://github.com/user-attachments/assets/96e0079f-a93a-4f5b-8809-748b15adf380) |